cspell.json

```diff
@@ -11,14 +11,15 @@
   "enableGlobDot": true,
   "dictionaries": ["project-words"],
   "ignorePaths": [
-    ".git",
-    "**/*.wasm",
```

Comment:
`examples/ink/erc20.wasm` is causing cspell to fail on CI because this was removed